backfill/2024-07-15-fenix_active_users_aggregates_v3/fenix_query.sql

```diff
@@ -0,0 +1,123 @@
+WITH distribution_id AS
+(
+  SELECT
+    client_info.client_id,
+    ARRAY_AGG(
+      metrics.string.metrics_distribution_id IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS distribution_id
+  FROM
+    `moz-fx-data-shared-prod.fenix.metrics` -- from 2021 until 2023-12-31
+    -- `moz-fx-data-shared-prod.fenix.baseline` -- from 2024-01-01
```

Comment:
What date range are we backfilling here? Should we break this down into two separate queries one for dates prior to 2024-01-01 and another one for dates 2024-01-01 and after to make it very explicit what the query looked like rather than relying on remembering to uncomment the line for specific dates?

Reply:
+1

Reply:
Queries separated with a comment in the header for the dates to backfill each.